src/lib/accessibility/Behaviors/Popup/popupBehavior.ts

```diff
@@ -33,6 +34,18 @@ const popupBehavior: Accessibility = (props: any) => {
         close: {
           keyCombinations: [{ keyCode: keyboardKey.Escape }],
         },
+        toggle: {
+          keyCombinations: _.includes(normalizedOn, 'click') && [
+            { keyCode: keyboardKey.Enter },
+            { keyCode: keyboardKey.Spacebar },
+          ],
+        },
+        open: {
+          keyCombinations: _.includes(normalizedOn, 'hover') && [
```

Comment:
this is misleading - the prop is about taking key combinations, but currently we are starting to handle pseudo states under the same key-combinations-devoted prop